addons/database-backup-and-maintenance.sh

```diff
@@ -57,9 +58,15 @@ else
     echo -e $BACKUP_DIRECTORY$BACKUP_PF_FILENAME ", file already created. \n"
 fi
 
+# check if MySQL or MariaDB is installed
+if mysql -V | grep "MariaDB"; then
+    $SQL_ENGINE='mariadb'
```

Comment:
This is a bug. 
It does not do what you think it does.
Shell variable assignment must not be prefixed with $.